versions.tf

```diff
@@ -1,11 +1,10 @@
 terraform {
-  required_version = ">= 1.0"
+  required_version = ">= 1.0.0"
```

Comment:
I don't think any of our other modules support `1.0.0` as the bare minimum. Only components have this version requirement.

https://github.com/search?q=org%3Acloudposse+%22%3E%3D+1.0.0%22+filename%3Aversions.tf&type=Code&ref=advsearch&l=&l=